docs/release/handbook.md

```diff
@@ -0,0 +1,288 @@
+- Ensure that future_git revisions_they provide only include bug fixes, and not new features, from the previously provided _git revision_
+- Declare expected common dependency version ranges
+- Declare a WG Liaison that will be communicating with the Release Team
+
+**Actions for Manifests WG:**
+
+- Get a git revision from all WGs, on the first day of the Feature Freeze period. WGs need to have a git revision ready to give to the manifests WG.
+- Create a tracking issue and keep track of the pending bug fixes for each WG as well as the currently provided _git revision_
+- Push a commit that updates the manifests for the different WGs, based on the _git revision_ they had provided.
+- Create a new **vX.Y-branch** branch in the manifests repo
+- Create a new **vX.Y-rc.1** tag that will be pointing to the branch created above
+- Add commits on top of the **vX.Y-branch** branch to include bug fixes. The other WGs can provide a new _git revision_ to Manifests WG leads. The new git revision must only include bug fixes, not new features.
+- On the last day of feature freeze, cut a new **vX.Y-rc.2** RC tag on the **vX.Y-branch** release branch.
+
+**Actions for the Release Team:**
+- Generate a changelog/feature summary for the community, to be presented at the community meeting
+    - This will allow WGs to highlight their work, and also act as a checkpoint for everyone to know that they can move to documentation and testing for the rest of the cycle.
+- Identify, early in the first week, features at risk. These should either be aggressively fixed or punted
+
+**Success Criteria:** All working group git branches created, manifests complete, features either have landed or been pushed to next release.
+
+### Manifests testing (1 week)
+
+The Manifests WG leads will be evaluating the final structure of manifests before Distribution owners start testing
+
+**Actions for other WGs:**
+- Provide updates and fixes for pending issues
+
+**Actions for Manifests WG:**
+- Ensure the manifests provide a deployable Kubeflow installation
+- Sync with other WGs on pending issues
+- After finished testing, create a new **vX.Y-rc.3** tag pointing at the commit of the **vX.Y-branch** release branch in case more bug fixes were cherry-picked
+
+**Success Criteria:** Manifests complete and fixes from working groups applied.
+
+### Documentation (3 weeks) (Starts the same time as Feature Freeze)
+
+Now that it’s been 3 weeks of fFeature freeze and continuous bug fixes we
+should all focus on ensuring that our docs are up to date, before distributions
+start testing. Features that are at risk or being worked up until the deadline
+can be documented last, ideally most of the feature work has landed already and
+those can start to be documented.
+
+**Actions for ALL WGs:** Help on keeping the docs up to date.
+
+**Actions for the Release Team:**
+- Request a list of features, from the Working Groups, that require updates to the documentation
```

Comment:
Even deprecations need documentation. Need to specify alternate ways to achieve similar or enhanced functionality.

Reply:
Good point Malini! Deprecations should also be documented. I'll add a bullet for this as well.

And indeed this information should also be part of release notes with each component release

Reply:
Added a commit for this